src/main/scala/it/scalalearn/calculator/Calculator.scala

```diff
@@ -0,0 +1,70 @@
+package it.scalalearn.calculator
+
+import scala.annotation.tailrec
+import scala.io.StdIn.readLine
+import scala.util.{Try, Success, Failure}
+
+/**
+ * Application for evaluating the numerical value of arithmetic expressions
+ */
+object Calculator {
+
+  /**
+   * Entry point: Print banner and run REPL
+   */
+  def main(args: Array[String]): Unit = {
+    println("ScalaCalc v0.1\n\n"
+      + "Enter an empty line or ctrl-D to quit (ctrl-Z + enter on Windows).\n\n"
+      + "Begin input with a single ? character to view the parse tree for the expression.\n")
```

Comment:
Nice feature in scala you have trippleqoote nearly like in python

```suggestion
    println("""
      | ScalaCalc v0.1
      |
      | Enter an empty line or ctrl-D to quit (ctrl-Z + enter on Windows).
      |
      | Begin input with a single ? character to view the parse tree for the expression.
      |""".stripMargin)
```

Reply:
Thanks, I didn't know about `.stripMargin`.